Gemfile.lock

```diff
@@ -199,8 +199,7 @@ GEM
     hashie (3.4.3)
     hike (1.2.3)
     hitimes (1.2.3)
-    hitimes (1.2.3-java)
-    http (0.9.8)
+    http (1.0.1)
```

Comment:
We need to fix the `http` version to `0.9.8`. The current `http` versions `>= 1.0.0` don't support `Celluloid::IO` which we need for non blocking IO in our watchers, see: https://github.com/httprb/http/wiki/Parallel-requests-with-Celluloid::IO